tools/SetupFlow/DevHome.SetupFlow/Views/DevDriveView.xaml

```diff
@@ -113,6 +113,7 @@
                     ValidationMode="InvalidInputOverwritten">
                 </NumberBox>
                 <ComboBox 
+                    AutomationProperties.Name="Size Unit"

```

Comment:
Just realized this but the "DevDriveWindowByteUnitComboBox" in the resource file is actually suppose to be here.  you could do the same thing I did for browse button:

Addition to StringResource.cs
https://github.com/microsoft/devhome/blob/a0f0bcfb3110853b08c3c8a3bed69f9b8cfe28fa/tools/SetupFlow/DevHome.SetupFlow/Services/StringResourceKey.cs#L21

Declaration:

https://github.com/microsoft/devhome/blob/a0f0bcfb3110853b08c3c8a3bed69f9b8cfe28fa/tools/SetupFlow/DevHome.SetupFlow/ViewModels/DevDriveViewModel.cs#L39

Definition:

https://github.com/microsoft/devhome/blob/a0f0bcfb3110853b08c3c8a3bed69f9b8cfe28fa/tools/SetupFlow/DevHome.SetupFlow/ViewModels/DevDriveViewModel.cs#L121

Property:

https://github.com/microsoft/devhome/blob/a0f0bcfb3110853b08c3c8a3bed69f9b8cfe28fa/tools/SetupFlow/DevHome.SetupFlow/ViewModels/DevDriveViewModel.cs#L237

The xaml:

https://github.com/microsoft/devhome/blob/a0f0bcfb3110853b08c3c8a3bed69f9b8cfe28fa/tools/SetupFlow/DevHome.SetupFlow/Views/DevDriveView.xaml#L174

In this case we'd just do 

```xaml
AutomationProperties.Name="{x:Bind ViewModel.DevDriveWindowByteUnitComboBox, Mode=OneTime}"
```

Feel free to change the resource string to match Size Unit, or keep it the same.

Reply:
Yes, that should be localized.  Thank you for catching that.

Reply:
Fixed.